src/sentry/search/snuba/executors.py

```diff
@@ -388,7 +386,7 @@ def query(
             # If the pre-filter query didn't include anything to significantly
             # filter down the number of results (from 'first_release', 'query',
```

Comment:
Nit: Maybe we should remove `query` from this documentation too

Reply:
Missed that, thanks